api/types/summarizer/summarizer.go

```diff
@@ -78,6 +78,17 @@ func ValidateInferenceModel(m *summarizerv1.InferenceModel) error {
 		if p.Openai.GetOpenaiModelId() == "" {
 			return trace.BadParameter("spec.openai.openai_model_id is required")
 		}
+	case *summarizerv1.InferenceModelSpec_Bedrock:
+		// TODO(bl-nero): Relax this condition once we implement spend controls.
+		if modules.GetModules().Features().Cloud {
```

Comment:
`modules.GetModules()` only works in the auth service process, but this code in the `api/` module could be used in other components, meaning this is not going to behave deterministically.

I think we'll need to enforce the cloud restriction somewhere in auth service code instead of in general-purpose code like this.

Reply:
Oh, wow. I would have never guessed. I'll add this to `GetModules()` documentation.